### PR TITLE
Fix/snap updates

### PR DESCRIPTION
**Pull request by sateeshperi**
- This PR introduces `lib` folder with `UTILS.groovy` with function to remove NXF version
- Adds nf-test version to CI job matrix

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/fetchngs/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/fetchngs _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: tests/pipeline/sra.nf.test [hunk 1]
```diff
@@ -2,7 +2,7 @@ nextflow_pipeline {
 
     name "Test Workflow main.nf"
     script "main.nf"
-    tag "pipeline"
+    tag "sra"
```
Comment (Emiller88) [inline tests/pipeline/sra.nf.test:5]: ¿Por Qué No Los Dos?

File: nf-test.config [hunk 2]
```diff
@@ -1,8 +1,16 @@
...
+
+    // location of library folder that is added automatically to the classpath
+    libDir "tests/pipeline/lib/"
+
+    // location of an optional nextflow.config file specific for executing tests
     configFile "nextflow.config"
-    profile "test,docker"
 
+    // run all test with the defined docker profile from the main nextflow.config
+    profile "test,docker"
```
Comment (Emiller88) [inline nf-test.config:15]: My usual soap box.

**Review by Emiller88 — APPROVED** [review]
Looks good!

File: tests/pipeline/sra.nf.test (hunk 1 above)
Comment (sateeshperi) [reply tests/pipeline/sra.nf.test:5]: of course! you got it :wink:

File: nf-test.config (hunk 2 above)
Comment (sateeshperi) [reply nf-test.config:15]: will move to using configs

**Comment by Emiller88** [comment]
Merged, since the only thing it changes were tests, it shouldn't break anything(in theory).

**Pull request merged**